src/main/webapp/js/query.spec.js

```diff
@@ -0,0 +1,333 @@
+/*=========================
+    RESTAURANT QUERY AND RE-ROLL
+ =========================*/
+describe("Test the query function", ()=>{
+    let dummyElement;
+    let htmlReponse;
+
+    beforeEach(() => {
+        htmlResponse = new Response('{\"pick\":{\"name\":\"Chipotle\",\"rating\":\"4.5\",\"photos\":[\"photo1\",\"photo2\"]},\"results\":[{\"name\":\"Qdoba\",\"rating\":\"4\",\"photos\":[\"photo1\",\"photo2\"]}],\"status\":\"OK\"}');
+        dummyElement = document.createElement('div');
+        spyOn(document, 'getElementById').and.returnValue(dummyElement);
+        spyOn(window, 'query').and.callThrough();
+        spyOn(window, 'resultsPage');
+    });
+
+    it("Tese the query funciton works properly", async () => {
+        spyOn(window, 'fetch').and.returnValue(Promise.resolve(htmlResponse));
+        let x = await query("string");
```

Comment:
do we need `let x`? it seems like we don't use this in any of the tests

Reply:
removed x